contrib/benitlux/src/client/android.nit

```diff
@@ -126,8 +126,7 @@ redef class ItemView
 	init do set_backgroud(native, app.native_context)
 
 	private fun set_backgroud(view: NativeView, context: NativeContext) in "Java" `{
```

Comment:
I'm just noticing this now, but there seems to be a somewhat irritating typo on this function's name